ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/flow_tracking_settings.schema.yml

```diff
@@ -9,43 +9,21 @@ keys:
   flow_tracking_settings:
     description: |-
       Define the flow tracking parameters for this topology.
+      By default, following config is created if flow tracking is enabled on any link
+      with flow_tracker_name set to FLOW-TRACKER
+      flow_tracking_settings:
+        hardware:
+          trackers:
+          - name: FLOW-TRACKER
+            record_export:
+              on_inactive_timeout: 70000
+              on_interval: 300000
+            exporters:
+            - name: CV-TELEMETRY
+              collector:
+                host: 127.0.0.1
+              local_interface: Loopback0
+              template_interval: 3600000
+          shutdown: false
```

Comment:
move this to the default: (it accepts a dict when the type is a dict and it will show up in the docs as a (+)

Reply:
Not sure this looks good. https://ansible-avd--3776.org.readthedocs.build/en/3776/roles/eos_designs/docs/wan-preview.html?h=flow+tracking##